src/logic/operate.js

```diff
@@ -1,18 +1,21 @@
-/* eslint-disable */
+// eslint-disable-next-line import/no-extraneous-dependencies
 import Big from 'big.js';
 
 const operate = (numberOne, numberTwo, operation) => {
+  const fNum = Big(numberOne);
+  const sNum = Big(numberTwo);
+
   let res = 0;
   if (operation === '+') {
-    res = Big(numberOne + numberTwo).toString();
+    res = (fNum + sNum).toString();
   } else if (operation === '-') {
-    res = Big(numberOne - numberTwo).toString();
+    res = (fNum - sNum).toString();
   } else if (operation === 'X') {
-    res = Big(numberOne * numberTwo).toString();
+    res = (fNum * sNum).toString();
   } else if (operation === '/') {
-    res = Big(numberOne / numberTwo).toString();
+    res = (fNum / sNum).toString();
```

Comment:
dont forget to manage DIVISION BY ZERO error on the next milestone, else it will generate an error